tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -3110,8 +3110,9 @@ lldbg_emit_local_variable(LL_DebugInfo *db, SPTR sptr, int findex,
       flags = 0;
     }
     var_mdnode = lldbg_create_local_variable_mdnode(
-        db, DW_TAG_auto_variable, blk_info->mdnode, symname, file_mdnode, 0, 0,
-        type_mdnode, flags, fwd);
+        db, DW_TAG_auto_variable, blk_info->mdnode, PASSBYVALG(sptr)
+       ? SYMNAME(MIDNUMG(sptr)) : symname, file_mdnode, 0, 0,
+                type_mdnode, flags, fwd, sptr);
```

Comment:
There seems to be more parameters here. Did you miss uploading the diff in the definition of lldbg_create_local_variable_mdnode or is there a typo here?

Reply:
Seems legit to me. Are you facing any problems while building this ?

Reply:
Isn't sptr an additional parameter here?

Reply:
Yes, I faced a build issue due to additional parameters.

Reply:
No. That may seem due to ternary operator inside this call.

Reply:
I checked it's building fine.

Reply:
Oops Thanks for this! I noticed Our down stream changes contain an extra `sptr`. Could you please put this PR review on hold.